index.ts

```diff
@@ -70,32 +70,54 @@ function sanitizeType(str: string): string | null {
  */
 function generateParameterDocumentation(functionNode: FunctionLikeDeclaration): void {
 	const params = functionNode.getParameters();
+
+	// Get param tag that matches the param
+	// IMPORTANT: change to `const jsDoc = getJsDocOrCreate(docNode);` when all PRs are merged
+	const jsDoc = getJsDocOrCreate(functionNode);
```

Comment:
This should be changed after https://github.com/futurGH/ts-to-jsdoc/pull/24 has been merged

```typescript
const jsDoc = getJsDocOrCreate(docNode);
```